measure_extinction/extdata.py

```diff
@@ -201,9 +198,11 @@ class ExtData:
     waves : dict of key:wavelengths
         key is BAND, IUE, IRS, etc.
 
-    ext : dict of key:E(lambda-v) measurements
+    ext : dict of key:E(lambda-v) or A(lambda)/A(V) measurements
 
-    uncs : dict of key:E(lambda-v) measurement uncertainties
+    uncs : dict of key:E(lambda-v) or A(lambda)/A(V) measurement uncertainties
+
+    stds : dict of key:A(lambda)/A(V) standard deviations (this is the spread of the sample around the population mean)
```

Comment:
Say something about this is only defined if the curve is an average of a set of curves?